extensions/adobe/experience/customerJourneyManagement/message-experienceevent.example.4.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
should we name the examples to indicate;
EVENT_TYPE & CHANNEL & STATUS

this one looks like feedback-push-exclude example for instance

Reply:
The current automation scripts which validates the examples with the corresponding schema file requires, example files to have a naming convention : schema-name.example.[0-9].json. Check line:
1. https://github.com/adobe/xdm/blob/master/test/example-valid.js#L13
2. https://github.com/adobe/xdm/blob/master/test/example-valid.js#L67